nimbus/src/updating.rs

```diff
@@ -0,0 +1,71 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+//! This module implements the primitive functions to implement
+//! safe updating from the server.
+
+use crate::error::Result;
+use crate::persistence::{Database, StoreId, Writer};
+use crate::Experiments;
+
+const KEY_PENDING_UPDATES: &str = "pending-experiment-updates";
+
+pub fn stash_pending_updates(db: &Database, experiments: Experiments) -> Result<()> {
```

Comment:
nit: not a huge fan of "stash" here when no other functions which write use it - and the term makes me think "oh, it's doing something write-like, but fancy?" - but it turns out it's just writing them. Contrast with `pop, which I think *is* fine - that name implies "read and remove" - to me stash implies "write and ???<profit?>" :)

OTOH, most other functions which do write don't explicitly use "write" or "save" etc in their name either - but it's worth considering.

Reply:
Renamed these to be more explicit.